workspace_integration_test.go

```diff
@@ -432,6 +432,50 @@ func TestWorkspacesCreate(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, options.TriggerPatterns, w.TriggerPatterns)
 	})
+
+	t.Run("when both speculative-enabled and pull-request-outputs-enabled are true (behind a feature flag)", func(t *testing.T) {
+		skipIfBeta(t)
```

Comment:
Let's not skip tests, but rather set the feature flag variant in `integration_flags.json`.
This file is read by the CI Atlas instance and in it you can define set of rules when to flip the feature flag on - e.g. if your organization name ends with `-ff-on`. ([an example](https://github.com/hashicorp/go-tfe/blob/6b5c37fbd622f39bd9784ff730f3595d213d58da/workspace_integration_test.go#L304-L332))

Reply:
I see. Thanks for letting me know!

Aside from using suffix like `-ff-on`, may I just blindly flip the feature on for every org with `tst-` prefix? Is it necessary to tests both on/off cases under the `tst-` orgs? I created a draft [PR](https://github.com/hashicorp/atlas/pull/12998/files) to update `integration_flags.json` and will update it accordingly.